src/vm/jitinterface.cpp

```diff
@@ -8912,6 +8912,72 @@ void CEEInfo::getMethodVTableOffset (CORINFO_METHOD_HANDLE methodHnd,
 }
 
 /*********************************************************************/
+CORINFO_METHOD_HANDLE CEEInfo::resolveVirtualMethod(CORINFO_METHOD_HANDLE methodHnd,
+                                                    CORINFO_CLASS_HANDLE derivedClass)
+{
+    CORINFO_METHOD_HANDLE result;
+
+    JIT_TO_EE_TRANSITION();
+
+    result = resolveVirtualMethodExact(methodHnd, derivedClass);
+
```

Comment:
I was matching jit interface changes from .Net Native. It looks like I may not need both versions.